_posts/2022-06-29-announcing-style-guide.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "OpenSearch style guidelines are now available"
+authors: 
+  - nbower
+date: 2022-06-29
+categories: 
+  - community
+---
+
+The OpenSearch team is excited to announce the release of the [style guidelines](https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md) for our documentation and marketing content! These guidelines cover the style standards and terms to be observed when creating OpenSearch content. Our growing team of technical writers and editors is committed to providing complete and best-in-class documentation to the OpenSearch community, and releasing our style guidelines reflects this commitment.
+
+Included in the style guidelines is guidance on voice and tone, punctuation and capitalization, numbers and measurement, formatting and organization, special considerations for blog posts, inclusive content, and more. We have also provided a list of [terms](https://github.com/opensearch-project/documentation-website/blob/main/TERMS.md) to be observed in OpenSearch content. These resources will evolve as we implement best practices and lessons learned from working with you, the community.
+
+As part of our focus on producing exceptional content, we are developing quality assurance processes, resources, and initiatives to help us standardize our approach to creating bar-raising OpenSearch content. Our style guidelines represent one of the mechanisms our team will use to achieve this. Check out the style guidelines, and feel free to let us know what you think.
+
+OpenSearch values transparency, and we welcome community contributions to our style guidelines. To access our documentation, see https://opensearch.org/docs/latest. If you want to contribute to the OpenSearch documentation, see the [CONTRIBUTING.md](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file, which covers how to open an issue or create a pull request on GitHub. We look forward to collaborating with you!
```

Comment:
"see https://opensearch.org/docs/latest."
should turn this into a live link

Reply:
Thanks Kris! @Naarcha-AWS I like both of Kris's suggestions.